apps/openmw/mwmechanics/character.cpp

```diff
@@ -1305,7 +1305,9 @@ bool CharacterController::updateWeaponState()
                 if(!resultMessage.empty())
                     MWBase::Environment::get().getWindowManager()->messageBox(resultMessage);
                 if(!resultSound.empty())
-                    MWBase::Environment::get().getSoundManager()->playSound(resultSound, 1.0f, 1.0f);
+                    MWBase::Environment::get().getSoundManager()->playSound3D(target,
+                    resultSound, 1.0f, 1.0f, MWBase::SoundManager::Play_TypeSfx,
+                    MWBase::SoundManager::Play_Normal);
```

Comment:
Note, sounds coming from the player are played in [2D by default](https://github.com/OpenMW/openmw/blob/master/apps/openmw/mwsound/soundmanagerimp.cpp#L554), even when you call playSound3D. They can't be played in 3D because we don't supply an accurate position of the sound source (which makes all the more difference when it's so close to the listener/camera).

So in practice, I don't think this change makes a difference, but there's no harm either.